pyanaconda/modules/boss/boss.py

```diff
@@ -132,3 +132,12 @@ def install_system_with_task(self):
         task = self._install_manager.install_system_with_task()
         path = self.publish_task(BOSS.namespace, task)
         return path
+
+    def set_locale(self, locale):
+        """Set locale of all modules.
+
+        :param str locale: locale to set
+        """
+        log.info("Setting locale of all modules to %s.", locale)
```

Comment:
Shouldn't the boss also call `super.set_locale()` to set up its own locale?

Reply:
Done